tests/explainers/test_tree.py

```diff
@@ -1205,13 +1058,175 @@ def test_HistGradientBoostingRegressor(self):
 
         # check that SHAP values sum to model output
         assert (
-            np.max(
-                np.abs(shap_values.sum(1) + explainer.expected_value - model.predict(X))
-            )
-            < 1e-4
+                np.max(
+                    np.abs(shap_values.sum(1) + explainer.expected_value - model.predict(X))
+                )
+                < 1e-4
         )
 
 
+class TestExplainerXGBoost:
+    """Tests for the TreeExplainer with XGBoost models.
+
+    Included models:
+        * XGBRegressor
+        * XGBClassifier
+    """
```

Comment:
XGBRanker is also tested in this TestClass

Reply:
Corrected by commit [d0c5987](https://github.com/shap/shap/pull/3192/commits/d0c59873e0949bb20a8ac42a690d44ebb22b6f45)